client/components/RelatedProducts/RelatedProducts.jsx

```diff
@@ -1,23 +1,56 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import RelatedProductsList from './Related/RelatedProductsList.jsx';
 import CustomOutfitList from './Outfit/CustomOutfitList.jsx';
 import Grid from '@material-ui/core/Grid';
-import products from './../DummyData/ProductDummyData';
+import axios from 'axios';
+import config from '../../../config.js'
 
+axios.defaults.headers.common['Authorization'] = config.GITHUB_TOKEN;
 
-const RelatedProducts = (id) => {
+const RelatedProducts = ({productId}) => {
```

Comment:
Minor nitpick, typically when you do destructuring you would surround it with spaces like ```{ productId }```

Reply:
Noted! Will do that in the future!